README.md

```diff
@@ -1 +1,116 @@
 # digital-media-object-processor
+The digital media processor objects can receive digital media object from two different sources:
+- Through the API as a request to register a digital media object
+- Through a Kafka queue to register an annotation
+
+## Preparation
+The digital media object processes the received object as a batch.
+To ensure that there are no conflicts we first ensure that the batch only contains unique objects.
+Objects which are updated twice in one batch are republished to the queue.
+After this we will collect the digital specimen PID from the digital specimen database.
+In the received data only the physical specimen id is available, so we need to collect the digital specimen id.
+If no digital specimen id is available in the database we will requeue the item as the specimen has not been processed yet.
+After this the system will evaluate if the received objects have are new, updated or equal to the current objects.
+To evaluate this the system takes the following steps:
+- Check if the object is already in the system, based on the properties:
+  - digital specimen id
+  - media url  
+We assume that these two properties make the digital media object unique and can be used as key.
+No single specimen could have a digital media object with the same media url.
+However, the same media url can be used with a different specimen (if multiple specimen are on the image).
+
+## Evaluation with existing objects
+We will now create three lists:
+- A list with new items when the digital media object cannot be found in the database
+- A list with updated items when a digital media object is found, but it differs from the newly received object
+- A list with equal items when the digital media object is found but is equal to the newly received object  
```

Comment:
A list with equal **the** items?

Reply:
Not sure here, it might not be grammatically correct, but I would rather keep the three things the same. A list with equal items doesn't sound that bad to me.